packages/dds/tree/package.json

```diff
@@ -31,8 +31,11 @@
     "eslint": "eslint --format stylish src",
     "eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
     "fence": "gf",
-    "lint": "npm run eslint",
-    "lint:fix": "npm run eslint:fix",
+    "format": "npm run prettier:fix",
+    "lint": "npm run prettier && npm run eslint",
+    "lint:fix": "npm run prettier:fix && npm run eslint:fix",
```

Comment:
I'd recommend splitting this PR up into a few PRs:

1. First add support for running prettier (the scripts below and adding the dependencies).
2. One or more PRs for various groups of files getting formatted (as long as the format does not conflict with the linter settings). Ex: a PR formatting all the markdown docs files, and one for all the package configuration files. If possible, include PRs formatting part of all of the actual source TS files as part of this (might be an issue if doing so conflicts with any of our linter rules.
3. As a final step, change the scripts here to change the linting and thus enforce the new formatting.

This has a few benefits: 
1. it makes the actual relevant changes in each step easier to review (ex: when reviewing initial support, I can tell if there are changes to the lint configuration that they are semantic and not just formatting)
2. Allows reviewers to be picked that have expertise in the relevant areas without also having lots of unrelated stuff they aren't knowledgeable about in the PR.
3. Reduces the risk and scope of conflicts, both for this PR and ones others are working on. It makes the only stuff likely to conflict the actual formatting which can always be resolved by just accepting the non-formatting version, then autoformatting later.
4. If there is a concurrent change which gets merged at the same time as the final enablement of the new formatting rules (which will break the build),  it would not be nearly as harmful to resolve (ex: it via a revert) since it would just touch a couple of script lines in package.json instead of code everything in the package.